src/lakefs_spec/spec.py

```diff
@@ -93,6 +93,38 @@ def parse(path: str) -> tuple[str, str, str]:
     return repo, ref, resource
 
 
+def ensure_branch(
+    client: LakeFSClient, repository: str, branch: str, source_branch: str = "main"
```

Comment:
```suggestion
    client: LakeFSClient, repository: str, branch: str, source_branch: str
```

Please drop the default, otherwise this might silently work when it really shouldn't